hydra-plutus/src/Hydra/Contract/Head.hs

```diff
@@ -226,15 +214,15 @@ checkCollectCom ctx@ScriptContext{scriptContextTxInfo = txInfo} (contestationPer
 
   extractAndCountCommits TxInInfo{txInInfoResolved} (commits, nCommits)
     | isHeadOutput txInInfoResolved =
-      (commits, nCommits)
+        (commits, nCommits)
     | hasPT headId txInInfoResolved =
-      case commitDatum txInfo txInInfoResolved of
-        Just commit@Commit{} ->
-          (commit : commits, succ nCommits)
-        Nothing ->
-          (commits, succ nCommits)
+        case commitDatum txInfo txInInfoResolved of
+          Just commit@Commit{} ->
+            (commit : commits, succ nCommits)
+          Nothing ->
+            (commits, succ nCommits)
     | otherwise =
-      (commits, nCommits)
+        (commits, nCommits)
```

Comment:
I would revert to the original code formatting so that we avoid future commit that will just reformat the old way because people are using the nix version of it.